frontend/src/services/apiService.ts

```diff
@@ -5,9 +5,17 @@ import { Page } from "../models/associations/page";
 import { Marketplace, Transaction } from "../models/associations/marketplace";
 import { Media } from "../models/associations/media";
 import { QueryResult, useQuery } from "react-query";
+import {Ballot, Election, Result} from "../models/associations/election";
+import {User} from "../models/user";
 
 const baseApi = "http://localhost:8000/api/v1";
 
+export enum activeStatus {
```

Comment:
Please give it a more precise name, like `ElectionStatus`.
Plus, according to the Typescript convention, the name of the enumeration should be capitalized.